projects/js-packages/publicize-components/src/components/media-section/styles.module.scss

```diff
@@ -0,0 +1,61 @@
+.wrapper {
+	padding: 0;
+
+	& :global( .components-spinner ) {
+		position: absolute;
+		top: 50%;
+		left: 50%;
+		margin-top: -9px;
+		margin-left: -9px;
+	}
```

Comment:
Similarly is there a reason why we need to adjust the spinner of the core component? This seems like a change that should be handled upstream.

Reply:
It's more of a mixing of core and custom components, we're using the core spinner component in our custom media picker component, which is why it needs some extra styling.